docs/developer/getting-started.md

```diff
@@ -47,11 +50,11 @@ Copy the `.env-template` file to `.env` and change the following settings:
 
 Check that the settings are correct by starting the server with `pnpm dev`.
 
-## API access
+:::tip
+Add `DB_DEV_SYSTEM_DEBUG_QUERY_LIMIT=500` and `DB_DEV_FOODS_DEBUG_QUERY_LIMIT=500` to the API `.env` file to limit debug query char limit. It can prevent long queries from cluttering the console. The API server runs food indexing each time it starts so the console outputs can be quite a lot. The server does that for all locales in the food database, so for the DEV instance, you can also limit number of locales it runs against in `.env` by setting `APP_ENABLED_LOCALES = ["UK_V2_2022"]`.
+:::
 
-The default user with superuser rights is `admin` and the password is `intake24`. Refer to the [API docs](/api/) for
-further details.
+## API access
 
-::: tip
-Use the e-mail log in endpoint (`/api/login`) for the admin user even though "admin" is not a valid e-mail address.
-:::
+Please add your email in DBeaver `users` tables, then go to `apps/cli`, and run `pnpm cli:dev hash-password yourNewPassword`.
```

Comment:
I would extend this a bit that user needs to create `user` and `user_password` records in database manually. Then add the password hash using the cli. This one only worked for you as I gave you snapshot where you had your accoult already created.

This DB setup bit is still a bit combersome as we don't have a clean way, it always depends if it's created from clean DB or some snapshot etc. I think we should have for this at least a CLI command that creates whole new account without this manual hassle to streamline this a bit.